sdl_android/src/main/java/com/smartdevicelink/transport/MultiplexTransportConfig.java

```diff
@@ -35,19 +72,26 @@
 	ComponentName service;
 	int securityLevel;
 
-	
+	List<TransportType> primaryTransports, secondaryTransports;
+	boolean requiresHighBandwidth = false;
+	TransportListener transportListener;
 
 	
 	public MultiplexTransportConfig(Context context, String appId) {
```

Comment:
There are a lot of public methods in this class with no javadocs

Reply:
A lot of methods were introduced prior to this PR but I will ad javadocs to all